specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json

```diff
@@ -0,0 +1,360 @@
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+      "/{resourceUri}/providers/microsoft.insights/metricBaselines": {
+        "get": {
+          "tags": [
+            "Baseline",
+            "Metric"
+          ],
+          "operationId": "MetricBaselines_Get",
+          "description": "**Gets the metric baseline values**.",
+          "x-ms-examples": {
+            "Get metric baselines": {
+              "$ref": "./examples/metricBaselines.json"
+            }
+          },
+          "parameters": [
+            {
+              "$ref": "#/parameters/ExtendedResourceUriParameter"
+            },
+            {
+              "$ref": "#/parameters/MetricNamesParameter"
+            },
+            {
+              "$ref": "#/parameters/MetricNamespaceParameter"
+            },
+            {
+              "$ref": "#/parameters/TimespanParameter"
+            },
+            {
+              "$ref": "#/parameters/IntervalParameter"
+            },
+            {
+              "$ref": "#/parameters/AggregationsParameter"
+            },
+            {
+              "$ref": "#/parameters/SensitivitiesParameter"
+            },
+            {
+              "name": "$filter",
+              "in": "query",
+              "type": "string",
+              "description": "The **$filter** is used to reduce the set of metric data returned.<br>Example:<br>Metric contains metadata A, B and C.<br>- Return all time series of C where A = a1 and B = b1 or b2<br>**$filter=A eq ‘a1’ and B eq ‘b1’ or B eq ‘b2’ and C eq ‘*’**<br>- Invalid variant:<br>**$filter=A eq ‘a1’ and B eq ‘b1’ and C eq ‘*’ or B = ‘b2’**<br>This is invalid because the logical or operator cannot separate two different metadata names.<br>- Return all time series where A = a1, B = b1 and C = c1:<br>**$filter=A eq ‘a1’ and B eq ‘b1’ and C eq ‘c1’**<br>- Return all time series where A = a1<br>**$filter=A eq ‘a1’ and B eq ‘*’ and C eq ‘*’**.",
+              "required": false
+            },
+            {
+              "$ref": "#/parameters/BaselineResultTypeParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            },
+            "200": {
+              "description": "Successful request to get the list of metric values.",
```

Comment:
is the operation returning a list?

Reply:
Yes, I updated the operation to indicate this is a List operation. I also updated some descrptions to indicate this.

Reply:
Ok, the response model doesn't look like a list, is that the proper response? Should the operation be x-ms-pageable? https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable